packages/components/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # next
```

Comment:
We either don't need this or should use the same header level `##` as a placeholder.